.eslintrc.json

```diff
@@ -2,7 +2,6 @@
   "root": true,
   "plugins": ["@typescript-eslint"],
   "rules": {
-    "no-console": ["error", { "allow": ["warn", "clear", "info", "error", "dir", "trace", "table"] }],
```

Comment:
Why are these removed? 🤓

Reply:
Honestly, just because I was tired of the warnings while I was debugging 😂